models.py

```diff
@@ -1109,15 +1114,15 @@ def __init__(self, *args, **kwargs):
 class FeatureForm(forms.Form):
 
   SHIPPED_HELP_TXT = ('First milestone to ship with this '
-                      'status. Applies to: Enabled by default, Behind a flag, '
-                      'Origin trial, Browser Intervention, and Deprecated. If '
+                      'status. Applies to: Enabled by default, In developer trial, '
+                      'Browser Intervention, and Deprecated. If '
                       'the flag is \'test\' rather than \'experimental\' set '
                       'status to In development.')
 
   SHIPPED_WEBVIEW_HELP_TXT = ('First milestone to ship with this status. '
                               'Applies to Enabled by default, Browser '
                               'Intervention, and Deprecated.\n\n NOTE: for '
-                              'statuses Behind a flag and Origin trial this '
+                              'statuses In developer trial and Origin trial this '
```

Comment:
Wouldn't this just go away?

Reply:
We still need that constant for the old UI.  However, since you pointed this out, I changed the help text for the new UI to just use SHIPPED_HELP_TXT for webview.  There is no need to tell the user to leave it blank because the field is not offered until they get past dev trials.